fetch/api/request/request-disturbed.html

```diff
@@ -48,6 +48,17 @@
         assert_throws(new TypeError(), function() { new Request(bodyConsumed); });
       }, "Check creating a new request from a disturbed request");
 
+      promise_test(function() {
+        assert_true(bodyConsumed.bodyUsed , "bodyUsed is true when request is disturbed");
+        const originalBody = bodyConsumed.body;
+        const bodyReplaced = new Request(bodyConsumed, { body: "Replaced body" });
+        assert_not_equals(bodyReplaced.body, originalBody, "new request's body is new");
+        assert_false(bodyReplaced.bodyUsed, "bodyUsed is false when request is not disturbed");
+        return bodyReplaced.text(text => {
```

Comment:
return bodyReplaced.text().then(text => {...});

Reply:
Whoa! You're right, will fix. There are multiple other instances of this in the other tests in this file -- will fix those too while I'm at it.

Reply:
Done.